app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -39,13 +40,27 @@ class Dashboard extends React.Component {
       if (this.state.display_course_summary) {
         // TODO
       } else if (this.state.assessment_type === 'GradeEntryForm') {
-        // TODO
+        $.get({url: Routes.grade_distribution_data_grade_entry_form_path(this.state.assessment_id)}).then(res => {
+
+          let new_labels = ['0 - 5']
```

Comment:
Similar to my comment on @devan-srinivasan's PR, don't generate the labels here. Instead, modify the controller method so that the labels are generated there.